lib/uplinkc/testdata_test.go

```diff
@@ -86,3 +92,65 @@ func TestC(t *testing.T) {
 		}
 	})
 }
+
+func TestLibstorj(t *testing.T) {
+	ctx := testcontext.NewWithTimeout(t, 5*time.Minute)
+	defer ctx.Cleanup()
+
+	libuplink := ctx.CompileShared(t, "uplink", "storj.io/storj/lib/uplinkc")
```

Comment:
Lets rename this variable for avoiding the collision with the `libuplink` package, rather than define a specific package identifier.
We recently added such convention https://github.com/storj/docs/blob/master/code/Style.md#variable-naming

Reply:
Good catch. I think we are trying to use the `libuplink` alias for the package at storj.io/storj/lib/uplink because there is a different `upliink` package at storj.io/storj/uplink. I will rename this variable, it's not a great name in addition to colliding.